shared/runtime/pyexec.c

```diff
@@ -64,7 +65,7 @@ static bool repl_display_debugging_info = 0;
 // EXEC_FLAG_ALLOW_DEBUGGING allows debugging info to be printed after executing the code
 // EXEC_FLAG_IS_REPL is used for REPL inputs (flag passed on to mp_compile)
 static int parse_compile_execute(const void *source, mp_parse_input_kind_t input_kind, mp_uint_t exec_flags) {
-    int ret = 0;
+    mp_int_t ret = 0;
```

Comment:
I think this `ret` variable should stay as `int`, since that's the return type of this function.